src/FakeClient/FakeClientMain.cpp

```diff
@@ -349,29 +385,50 @@ int main(int argc, char* argv[]) {
     }
   }
 
-  if (absl::GetFlag(FLAGS_frame_time)) {
+  const bool calculate_frame_time = absl::GetFlag(FLAGS_frame_time);
+  LOG("frame_time=%d", calculate_frame_time);
+  if (calculate_frame_time) {
     // Instrument vkQueuePresentKHR, if possible.
```

Comment:
This comment needs to be updated. E.g.:
"We want to instrument vkQueuePresentKHR, but some applications don't call libVulkan library directly..." etc.